prime-router/metadata/schemas/WATERS/waters-covid-19.schema

```diff
@@ -74,8 +74,19 @@ elements:
   - name: patient_race
     csvFields: [{ name: patientRace}]
 
+  # ReportStream uses hl70189 (H,N,U), but Waters sends HHS (2135-2, 2186-5, UNK, ASKU)
+  # Simplest way to map them was with an alt table
   - name: patient_ethnicity
-    csvFields: [{ name: patientEthnicity}]
+    altValues:
+      - code: H
+        display: 2135-2
+      - code: N
+        display: 2186-5
+      - code: U
+        display: UNK
+      - code: U
+        display: ASKU
+    csvFields: [{ name: patientEthnicity, format: $alt}]
 
```

Comment:
I fear that having to same code for "U" for both UNK and ASKU will lead to problems. 

There appears to be OMB vs ELR community argument about what is the right value set to use. PHINVADS and the confluence board should be the source of truth. From the notes, on the confluence board, the recommendation may expand to include

Reply:
ReportStream currently uses hl70189 as its INTERNAL representation, which has only 3 values.

Waters sends omb_ethnicity, which has four values.    Waters is probably right.  We should switch covid-19.schema to use omb_ethnicity.

But that means a ton of changes both inbound and outbound, that I'm not ready to work on right now.  This is a reasonable workaround.    For incoming data, mapping both UNK and ASKU is the best we can do --- there's no way to drop from 4 values to 3 without a loss of information.

Reply:
Right. Why don't we move to OMB for our internal?

Reply:
I agree that's the right approach, but we'd need to carefully test that both upstream and downstream data are not affected.    At the moment, let's get the waters schema out the door and working.